### ISSUE TITLE
Linebreak errors with eslint

### ISSUE THREAD

**Issue by Owen-Mak** [humphd/bridge-troll#7]
I ran eslint in Windows and it generated a list of errors regarding line-break style. (see screenshot below).  

![image](https://user-images.githubusercontent.com/25869561/37416352-e6ce81b6-2783-11e8-81af-73aa507591dc.png)

Steps to reproduce:
1. run `npm run eslint`

Expected result:
No errors reported, but possible warnings.

Possible fix:
Following some [research](https://eslint.org/docs/rules/linebreak-style), I think the cause of the issue is from [.eslintrc.yml  ](https://github.com/humphd/bridge-troll/blob/master/.eslintrc.yml).  For windows users, we need to change the linebreak style from unix to windows. 

I could not find any [environment variables](https://eslint.org/docs/user-guide/configuring#specifying-environments) for eslint to detect the current operating system, which would allow users to run a separate set of rules for different OS.

Node version: 8.9.4
OS: Windows 7

**Comment by humphd** [comment]
try running `npm run prettier` and it will fix it.  Then change your editor to use `LF` for these files.

**Comment by humphd** [comment]
I also wonder if we should remove the `LF` eslint error, since Prettier will just fix them?

**Comment by Owen-Mak** [comment]
I tried `npm run prettier` and got the following error:

![image](https://user-images.githubusercontent.com/25869561/37494872-126a13dc-2881-11e8-83cb-b567602e0c6f.png)

I have attached the error log in to this message since I am not sure the cause.

[2018-03-15T22_46_23_044Z-debug.log](https://github.com/humphd/bridge-troll/files/1817349/2018-03-15T22_46_23_044Z-debug.log)

**Comment by humphd** [comment]
@Owen-Mak, can you please try changing the `package.json` file line 9 and 10 to this (uses `\"` and `\"` around the file glob pattern:
```
    "prettier": "prettier --single-quote --write \"{tools,src,test}/**/*.{js,css}\"",
```

See https://stackoverflow.com/questions/48078281/run-prettier-on-windows.  If that works, can you please send a Pull Request with the change?

**Comment by Owen-Mak** [comment]
The fix worked for windows and the prettier commands now work.
It is filed under [PR#12](https://github.com/humphd/bridge-troll/pull/12)

However, after running `npm run prettier` successfully, the issues still remains when I run `npm run eslint`.
![image](https://user-images.githubusercontent.com/25869561/37500252-d2460cf2-289e-11e8-969e-fa511b23fc94.png)

**Comment by humphd** [comment]
@Owen-Mak can you look at https://help.github.com/articles/dealing-with-line-endings/#platform-windows and make sure you have `git`'s line endings configured properly?

You can do `git checkout -- tools/csv2json.js` if you need to reset that.

**Comment by humphd** [comment]
Fixed by https://github.com/humphd/bridge-troll/pull/15.

@Owen-Mak a tip for you: when you make pull requests, use a description message like this: `Fix #7: ...` and GitHub will automatically close the issue #7 when that PR is merged.
